src/common/OsmtAssertException.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022, Antti Hyvarinen <antti.hyvarinen@gmail.com>
+ * Copyright (c) 2022, Masoud Asadzade <masoud.asadzade@gmail.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#ifdef OSMT_ASSERT_EXCEPTION_H
+#else
```

Comment:
```suggestion
#ifndef OSMT_ASSERT_EXCEPTION_H
```

Reply:
From SMTS perspective,  It's working. Actually, SMTS'client just relies on OpenSMT to throw exceptions at asserts, so it catches them by base exception, so SMTS's client wouldn't need to include asserts.